src/libs/Middleware/SaveResponseInOnyx.js

```diff
@@ -0,0 +1,32 @@
+import Onyx from 'react-native-onyx';
+import _ from 'underscore';
+
+/**
+ * @param {Promise} response
+ * @param {Object} request
+ * @returns {Promise}
+ */
+function SaveResponseInOnyx(response, request) {
+    return response
+        .then((responseData) => {
+            // We'll only save the onyxData, successData and failureData for the refactored commands
+            if (_.has(responseData, 'onyxData')) {
```

Comment:
Hey, @marcochavezf! I don't think all refactored commands will return `onyxData` in the response. Some commands will return the response via pusher (i.e. `$onyxUpdate`) but we still need the middleware to save the `successData` and `failureData` in Onyx. Should we update this logic to account for this?

Example [here](https://github.com/Expensify/Web-Expensify/pull/33995/files#diff-8367f109728fcc54db479ccc765a52d2115e52d35eba9c660e665be1a34411acR1445-R1459). 

cc: @marcaaron

Reply:
@MariaHCD I'm accounting for this in the PR here: src/libs/Middleware/SaveResponseInOnyx.js